sequencer/src/block/payload.rs

```diff
@@ -110,10 +107,10 @@ impl<TableWord: TableWordTraits> Payload<TableWord> {
         &self,
         ns_table: &NameSpaceTable<TxTableEntryWord>,
         ns_id: VmId,
-        vid: &VidScheme,
-        vid_common: <VidScheme as VidSchemeTrait>::Common,
+        vid: &VidSchemeType,
```

Comment:
We could consider removing this as an argument since it can be constructed from common

Reply:
True, but VID scheme construction currently has nontrivial costs. It would be nice to avoid unnecessary VID scheme construction. OTOH if the caller doesn't already have a VID scheme then we're forcing her to make one. Ok I'll remove it.

Reply:
done in https://github.com/EspressoSystems/espresso-sequencer/pull/1169/commits/dac7e66b0b39ba1f35864b599d4deb868940f277